codecov_cli/services/staticanalysis/__init__.py

```diff
@@ -183,11 +184,15 @@ async def process_files(
     all_data = {}
     file_metadata = []
     errors = {}
+    if sys.platform.startswith("win32") or sys.platform.startswith("darwin"):
```

Comment:
Add a comment with link to https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods please